tensorflow/tensorflow.bzl

```diff
@@ -903,18 +904,52 @@ def tf_cuda_cc_test(
             "//conditions:default": 0,
         }),
         suffix = "_gpu",
-        tags = tags + tf_cuda_tests_tags(),
-        deps = deps + if_cuda([
+        tags = tags + tf_gpu_tests_tags(),
+        deps = deps + if_cuda_is_configured([
+            clean_dep("//tensorflow/core:gpu_runtime"),
+        ]) + if_rocm_is_configured([
             clean_dep("//tensorflow/core:gpu_runtime"),
         ]),
     )
 
+register_extension_info(
+    extension_name = "tf_gpu_cc_test",
+    label_regex_for_dep = "{extension_name}",
+)
+
+# terminology changes: saving tf_cuda_* definition for compatibility
+def tf_cuda_cc_test(
+        name,
+        srcs = [],
+        deps = [],
+        tags = [],
+        data = [],
+        size = "medium",
+        extra_copts = [],
+        linkstatic = 0,
+        args = [],
+        kernels = [],
+        linkopts = []):
+    tf_gpu_cc_test(
+        name,
```

Comment:
instead of an exhaustive copy, let's rewrite this rule as this:

```
tf_cuda_cc_test(*args, **kwargs):
    tf_gpu_cc_test(*args, **kwargs)
```

Reply:
Yep, good idea.  Thanks for the suggestions.